tf/do/variables.tf

```diff
@@ -0,0 +1,855 @@
+variable "do_token" {
+  description = "Your DigitalOcean API token"
+  default     = "da616614ccb17b31ef8c0cf993f36885ea084001d626926e1a4ea75f7da68d66"
```

Comment:
man, this is huge security issue! never expose any token or key to github PR.  Please change your token on DO, and only save it locally, and use some environment variable or cli parameter to pass in to the recipe.

Reply:
for testing only, it has been deleted.